app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -332,30 +328,26 @@ ShortFormApplicationController = (
   $scope.getLandingPage = (section) ->
     ShortFormNavigationService.getLandingPage(section)
 
-  $scope.goToLandingPage = (section) ->
```

Comment:
I wonder if this would be clearer if we called it goToSection, and then have a docstring that says "go to first page in section".

Looking at it it took me a while to figure out what a landing page is